### PR TITLE
Optional argument for `variation()`

### PR DESCRIPTION
**Pull request by itsdebartha**
- Added `corrected=true/false` for variation
- Added test for the above addition

This commit addresses #777

### CODE REVIEW

**Comment by ParadaCarleton** [comment]
@nalimilan do you know why the docs are failing?

File: src/scalarstats.jl [hunk 1]
```diff
@@ -411,7 +411,7 @@ a precomputed mean `m`. The coefficient of variation is the ratio of the
 standard deviation to the mean.
 """
 variation(x, m) = stdm(x, m) / m
```
Comment (devmotion) [inline src/scalarstats.jl:413]: This should be changed as well, I assume:
```suggestion
variation(x, m; corrected::Bool=true) = stdm(x, m; corrected=corrected) / m
```

**Review by devmotion — COMMENTED** [review]
Can you also update the docstring?

File: src/scalarstats.jl (hunk 1 above)
Comment (itsdebartha) [reply src/scalarstats.jl:413]: I found `stdm` to be in `deprecetes.jl` as of November 2021. So I was wondering if I need to change it or not...

Comment (devmotion) [reply src/scalarstats.jl]: Only specific methods of `stdm` with weights are deprecated. The function that is called here is defined in the Statistics (https://docs.julialang.org/en/v1/stdlib/Statistics/#Statistics.stdm) and not (yet) deprecated there (https://github.com/JuliaStats/Statistics.jl/issues/5).

Comment (itsdebartha) [reply src/scalarstats.jl]: Alright. I have made the required changes. Thanks.

**Comment by itsdebartha** [comment]
I also added tests that would use the method of `stdm` (i.e. specified mean) which were previously not there

**Comment by ParadaCarleton** [comment]
This looks good, thanks!

**Comment by CarloLucibello** [comment]
tag a new release? it has been a while...

**Pull request merged**